lib/api/controllers/indexController.js

```diff
@@ -161,7 +161,8 @@ class IndexController extends BaseController {
    * @returns {Promise<Object>}
    */
   refreshInternal() {
-    return this.kuzzle.internalEngine.refresh()
+    // @todo remove refresh* methods
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)